auditwheel/wheel_abi.py

```diff
@@ -116,7 +117,65 @@ def analyze_wheel_abi(wheel_fn: str):
     log.debug('external reference info')
     log.debug(json.dumps(external_refs, indent=4))
 
+    # get versioned symbols from external_refs
+    # {realpath: libname} e.g.
+    # {'/path/to/external_ref.so.1.2.3': 'external_ref.so.1'}
+    external_ref_no_linux = {}
+    # go through all policies
+    for e in external_refs.values():
+        # linux tag (priority 0) has no white-list, do not analyze it
+        if e['priority'] == 0:
+            continue
+        # go through all libs, retrieving their soname and realpath
+        for libname, realpath in e['libs'].items():
+            # we can't analyze the lib if we don't know where it's located
+            if realpath is None:
+                continue
+            if realpath not in external_ref_no_linux.keys():
+                external_ref_no_linux[realpath] = libname
+    # {libname: {depname: set([symbol_version])}} e.g.
+    # {'external_ref.so.1': {'libc.so.6', set(['GLIBC_2.5','GLIBC_2.12'])}}
+    external_refs_symbol_policy = {}
```

Comment:
This should be another function, returning the `external_refs_symbol_policy` dict.